PE extern weak symbols do not work like ELF, and fail to link on x86-64.

--- origsrc/gxine-0.5.906/src/utils.c	2010-12-05 05:38:44.000000000 -0600
+++ src/gxine-0.5.906/src/utils.c	2011-12-11 15:37:18.075465200 -0600
@@ -797,8 +797,6 @@ void config_update_default (const char *
 
 /* List a subset of video plugins */
 
-extern const char *const *xine_list_video_output_plugins_typed (xine_t *, uint64_t)
-	__attribute__((__weak__));
 
 const char *const *gxine_list_video_output_plugins (xine_t *xine)
 {
